packages/alfa-dom/test/tables/structure.spec.tsx

```diff
@@ -0,0 +1,51 @@
+import { jsx } from "@siteimprove/alfa-dom/jsx";
+import { test } from "@siteimprove/alfa-test";
+import {Slot, rowProcessing, processRowGroup, formingTable, Element, newTable} from "../../src";
+import {apple, complexRow, expenses, expensesNum, rowGroup, simpleRow, smithonian} from "./testcases";
+
+const cleanElement = (element: Element) =>
+  element
+    .attribute("id")
+    .map(attribute => attribute.value)
+    .getOr("");
+
+
+const cleanSlot = (slot: Slot) => (
+  {...slot,
+    elements: slot
+      .elements
+      .map(cleanElement)
+  }
+);
```

Comment:
Pure debug helpers to `console.dir` the table without showing the full elements…